.travis.yml

```diff
@@ -52,21 +48,19 @@ matrix:
       python: 3.5
     - env: TOX_SUFFIX="requests1"
       python: 3.6
-    - env: TOX_SUFFIX="aiohttp"
-      python: 2.6
     - env: TOX_SUFFIX="aiohttp"
       python: 2.7
     - env: TOX_SUFFIX="aiohttp"
       python: pypy
     - env: TOX_SUFFIX="aiohttp"
-      python: pypy3
+      python: "pypy3.5-5.9.0"
```

Comment:
<del>this pypy3 should be pypy3.5-5.9.0</del>

I'm dumb. This is under exclude. and we don't want to exclude pypy3.5 from aiohttp tests.

Reply:
OK, this is back to how it was :)